gui/control.js

```diff
@@ -17,13 +17,44 @@ const addAccountButton = document.getElementById('add-account-btn');
 window.addEventListener('unload', beforeQuit)
 window.addEventListener('DOMContentLoaded', main)
 
+const loadingMessage = document.getElementById('loading-message');
+const loadingOverlay = document.getElementById('loading-overlay');
```

Comment:
move this inside runWithLoading function

Reply:
Have removed loading message as there was no need for it. Moved loading overlay message inside loadingOverlay.